integration_tests/tests/match/placementApplications.cy.ts

```diff
@@ -237,6 +242,151 @@ context('Placement Applications', () => {
     })
   })
 
+  it('allows me to review an application', () => {
+    // Given there is a placement request task and placement application in the database
+    const placementApplicationTasks = placementApplicationTaskFactory.buildList(1)
+
+    const document = {
+      'request-a-placement': [{ 'test question 1': 'test answer 1' }, { 'test question 2': 'test answer 2' }],
+    }
+    const placementApplication = placementApplicationFactory.build({ id: placementApplicationTasks[0].id, document })
+    cy.task('stubPlacementApplication', placementApplication)
+
+    cy.task('stubTasks', placementApplicationTasks)
+
+    // When I visit the placementRequests dashboard
+    const listPage = ListPage.visit()
+    listPage.clickPlacementApplications()
+
+    // And I click on the first name
+    cy.get('a').contains(placementApplicationTasks[0].person.name).click()
+
+    // Then I should be taken to the review applications page
+    const page = Page.verifyOnPage(ReviewApplicationPage)
+
+    cy.get(`a[href="${paths.applications.show({ id: placementApplication.applicationId })}"]`).contains(
```

Comment:
It would be to extract these to methods on the page so we can describe them better